app/Http/Controllers/GetArticleController.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers;
+
+use Acme\Article\Article;
+use Acme\Article\Repository\Exception\ArticleNotFound;
+use Acme\Article\UseCase\GetArticle\GetArticleCommand;
+use Acme\Article\UseCase\GetArticle\GetArticleHandler;
+use Acme\Article\ValueObject\ArticleID;
+
+class GetArticleController extends Controller
+{
+    /**
+     * @var GetArticleHandler
+     */
+    private $handler;
+
+    public function __construct(GetArticleHandler $handler)
+    {
+        $this->handler = $handler;
+    }
+
+    public function __invoke(string $id)
+    {
+        $command = new GetArticleCommand(ArticleID::fromUUID($id));
+
+        try {
+            $article = $this->handler->__invoke($command);
+        } catch (ArticleNotFound $e) {
```

Comment:
Please add a Logger to log the exception.
This will be helpful when sth will fail.

Reply:
I have in mind a different way to manage the errors. This was just the fastest solution to make the job done

Reply:
You need to log anyway the exception when it happens.
If you want to add in the last handler the exception may be caught and re-thrown with a different one, so you're losing it.
It's up to you add it here or in the repository with a proper log level.